src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/SdkDependencyModel.cs

```diff
@@ -43,5 +43,19 @@ internal class SdkDependencyModel : DependencyModel
                                 .FirstOrDefault();
             Caption = string.IsNullOrEmpty(Version) ? baseCaption : $"{baseCaption} ({Version})";
         }
+
+        private string _id;
+        public override string Id
+        {
+            get
+            {
+                if (_id == null)
```

Comment:
Is this really worth it?  It looks like `OriginalItemSpec` is passed to the base already constructed.  Does the property do anything other than just return it?